README.md

```diff
@@ -126,7 +126,7 @@ You can also install AllenNLP by cloning our git repository:
 Create a Python 3.6 virtual environment, and install the necessary requirements by running:
 
   ```bash
-  scripts/install_requirements.sh
+  pip install -r requirements.txt
```

Comment:
I'm actually not sure that we need this anymore, because `pip install --editable .` might just work.  But I don't know enough about this set up to be the right person to make that call.  @schmmd, any ideas?

Reply:
This was a user contribution and I don't fully understand it--but it looks like you're right.

```
root@4ca47cb5bdf7:/allennlp# pip install --editable .
Obtaining file:///allennlp
Collecting torch>=0.4.1 (from allennlp===0.8.3-unreleased)
  Downloading https://files.pythonhosted.org/packages/31/ca/dd2c64f8ab5e7985c4af6e62da933849293906edcdb70dac679c93477733/torch-1.0.1.post2-cp36-cp36m-manylinux1_x86_64.whl (582.5MB)
...
```